javascript/operators/nullish_coalescing_operator.json

```diff
@@ -0,0 +1,59 @@
+{
+  "javascript": {
+    "operators": {
+      "optional_chaining": {
+        "__compat": {
+          "description": "Nullish coalescing operator (<code>??</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator",
+          "spec_url": "https://tc39.es/proposal-nullish-coalescing/#top",
```

Comment:
Maybe this?

```suggestion
          "spec_url": "https://tc39.es/proposal-nullish-coalescing/#sec-conditional-operator",
```

Reply:
Well, I'm not sure that pointing to the grammar of the ternary operator is helpful.
By default, I'd suggest https://tc39.es/proposal-nullish-coalescing/#sec-intro instead.

I don't know if this topic is tracked elsewhere but when submitting, having to put a fragment to validate the schema seemed weird to me (at least for this case where a proposal targets a single feature / I reckon it's helpful in a longer/richer spec).

Reply:
Yeah, we're discussing currently whether requiring fragments make sense. At the point this lands in the main spec, it does make sense to have a good fragment id, but right now it doesn't really.

Reply:
#top works for now. Thank you!